modules/swagger-codegen/src/main/resources/swift3/Configuration.mustache

```diff
@@ -0,0 +1,11 @@
+// Configuration.swift
+//
+// Generated by swagger-codegen
+// https://github.com/swagger-api/swagger-codegen
+//
+
+import Foundation
+
+open class Configuration {
+    open static var dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSSZZZZZ"
```

Comment:
I still don't really think this is a good idea...and you will be changing the date format for your entire client. Also should be very careful to note that changing this value after it is read in the constructor of the formatter will have no effect. 

I recommend you always use ISO-8601 formatted datetimes for encoding dates, and deal with any transformations you desire elsewhere. 

https://github.com/swagger-api/swagger-codegen/issues/5318#issuecomment-291525782